payjoin-cli/src/db/v2.rs

```diff
@@ -0,0 +1,55 @@
+use bitcoincore_rpc::jsonrpc::serde_json;
+use payjoin::receive::v2::Enrolled;
+use payjoin::send::RequestContext;
+use sled::IVec;
+
+use super::*;
+
+impl Database {
+    pub(crate) fn insert_recv_session(&self, session: Enrolled) -> Result<()> {
+        let key = &session.public_key().serialize();
+        let value = serde_json::to_string(&session).map_err(Error::Serialize)?;
+        self.0.insert(key.as_slice(), IVec::from(value.as_str()))?;
+        self.0.flush()?;
+        Ok(())
+    }
+
+    pub(crate) fn get_recv_session(&self) -> Result<Option<Enrolled>> {
+        if let Some(ivec) = self.0.get("recv_sessions")? {
```

Comment:
Do we ever insert an entry with "recv_sessions" as the key? It looks to me like this always returns `None`. Likewise for "send_sessions".